app/shop.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+
+
+class Shop:
+    def __init__(
+        self,
+        name: str,
+        location: list,
+        products: dict
+    ) -> None:
+        self.name = name
+        self.location = location
+        self.products = products
+
+    def take_a_check(self, list_of_customer_param: list) -> None:
+        print(f"Date: {datetime.datetime.now().strftime('%d/%m/%Y %H:%M:%S')}")
+        print(f"Thanks, {list_of_customer_param[0]}, for your purchase!")
+        print("You have bought: ")

```

Comment:
Remove space after the :
```suggestion
        print("You have bought:")

```